src/Widgets/NetworkWidget.vala

```diff
@@ -39,15 +39,11 @@ namespace WingpanelMonitor {
             upload_label.set_width_chars (8);
             upload_label.halign = Gtk.Align.START;
             var upload_label_context = upload_label.get_style_context ();
-            upload_label_context.add_class ("small-label");
```

Comment:
Can you please explain why have you removed this classes?

Reply:
ignore this commit 😅️